CurrentTime.js

```diff
@@ -0,0 +1,32 @@
+function showTime(){
+    let date = new Date();
+    let tm = [
+        date.getHours(), 
+        date.getMinutes(), 
+        date.getSeconds(),
+        date.toLocaleString('default', {weekday: 'long'}), 
+        date.getDate(),
+        date.toLocaleString('default', {month: 'long' })];
+
+    for (let i = 0; i < 3; i++) {
+        if (tm[i] < 10) {
+            tm[i] = "0" + tm[i].toString();
+        }
+        else {
+            tm[i] = "" + tm[i].toString();
+        }
+    }
```

Comment:
There is `.padStart(...)` method for this case.
Also try to avoid mixed-type arrays, so it make sense to coerce all values to String.

```suggestion
    const tm = [
        date.getHours().toString().padStart(2, '0'), 
        date.getMinutes().toString().padStart(2, '0'), 
        date.getSeconds().toString().padStart(2, '0'),
        date.toLocaleString('default', {weekday: 'long'}), 
        date.getDate().toString(),
        date.toLocaleString('default', {month: 'long' })
    ];
```